modules/ROOT/pages/organization.adoc

```diff
@@ -19,47 +19,63 @@ An Anypoint Platform user who has been granted organization administrator privil
 [NOTE]
 Business groups are not enabled by default in a new Anypoint Platform account. To activate business groups in your organization, contact your MuleSoft representative.
 
+== Access the Organization Administration
```

Comment:
This comment is for line 7/8, which GitHub wouldn't let me "click." I would expect this topic to start with a definition of Organization, then launch into whom you have to be to work on it, so something like:

An Anypoint Platform organization is the [what is this? how different from an acct?]. 

Then put the Org admin info before or after Org Owner with its own heading.

Reply:
The title in line 22 confuses me. Access as in Access Mgmt? Access as in regular old meaning of getting into an account? I'd be tempted to call this "Logging into an Org" or similar. This just seems an odd turn of phrase to me. (I could be wrong!) I don't really know what an org is for.

Reply:
Also there's a typo in line 7 "Organziation"